### DISCUSSION TITLE
how can i configure agent for apche solr

### DISCUSSION THREAD

**Discussion by kannan84240** [apache/skywalking#10227] [General]
how to configure and monitor apache solr plz share the document link or suggest me.

**Comment by kannan84240** [comment]
any one dont know?

**Reply by pg-yang** [reply]
If you mean to enhance apache solr client, you could refer to https://skywalking.apache.org/docs/skywalking-java/next/en/setup/service-agent/java-agent/java-plugin-development-guide/ to confirm if it's supported.
Also, you could refer to https://skywalking.apache.org/docs/skywalking-java/next/en/setup/service-agent/java-agent/java-plugin-development-guide/ to develop your own plugins.
If you want to monitor apache solr server, OAP could receive OTEL data and calculate metrics by MAL. Refer to https://skywalking.apache.org/docs/main/next/en/setup/backend/opentelemetry-receiver/

**Reply by kannan84240** [reply]
oh ok thanks @pg-yang
